### ISSUE TITLE
http://www.shqiptvlive.net

### ISSUE THREAD

**Issue by ItsBeno** [reek/anti-adblock-killer#3217]
http://imgur.com/a/slL75

Firefox 
Greasemonkey
AdGuard AdBlocker
AakList

**Comment by uBlock-user** [comment]
uBO - 

```
albtvhd.com,tvshqiphd.com##body > *[id^="blockblock"]:style(visibility: visible !important; display: block !important;)
albtvhd.com,tvshqiphd.com##table[id^='blockblock']

```

**Comment by jspenguin2017** [comment]
I can't reproduce it. 
![image](https://cloud.githubusercontent.com/assets/7283682/26210909/5c932a36-3bae-11e7-86f3-2a1a8ceac681.png)

**Comment by uBlock-user** [comment]
Try  - http://www.shqiptvlive.net/p/bigbrotheralbania9-live.html

**Comment by jspenguin2017** [comment]
Eh... In that case your rules don't seem to work... 
![image](https://cloud.githubusercontent.com/assets/7283682/26217483/c9c7b228-3bc4-11e7-8b85-abb96937b308.png)

**Comment by ItsBeno** [comment]
what is uBO- ?... sorry for this question ... uBlock Origin ?

**Comment by jspenguin2017** [comment]
Yes

**Comment by uBlock-user** [comment]
@jspenguin2017 Yes, it works. Add the relevant iframe domain before the `##`

The anti adblock screen comes from an iframe of the original domain and the iframe domain changes with every channel you open, so add the domain of the iframe, not the domain of the website.

For example, the link I gave you in https://github.com/reek/anti-adblock-killer/issues/3217#issuecomment-302485061 will have these filters

```
shqiperiatv.com,albtvhd.com,tvshqiphd.com##body > *[id^="blockblock"]:style(visibility: visible !important; display: block !important;)
shqiperiatv.com,albtvhd.com,tvshqiphd.com##table[id^='blockblock']
```
**shqiperiatv.com** needs to be added for the link I provided, so for the rest of the links, find the domain of their iframes individually and add them all together to patch them all.

**Comment by ItsBeno** [comment]
working perfect ! Thanks !

**Comment by jspenguin2017** [comment]
@uBlock-user OK... But they have like 50 channels...

**Comment by uBlock-user** [comment]
@jspenguin2017 Yeah that's going to be a pain collecting them individually or if you can cook up a script that can collect all those domains, that will save the time.

**Comment by jspenguin2017** [comment]
@uBlock-user Definitely, I will not manually collect them, only a slight problem is that we need to navigate to each page in order to grab the domain of the iframe... Probably need more than Userscript to do this... I'm thinking Node.js.

**Comment by ItsBeno** [comment]
@uBlock-user on each live stream website i have to add the iframe domain before the `## ` and it will be fixed or each website has another script ?

**Comment by uBlock-user** [comment]
> on each live stream website i have to add the iframe domain

yes

**Comment by jspenguin2017** [comment]
The `iframe` is directly in the code 😄 
I will collect all 71 of the `iframes` with a Node.JS script.

**Comment by ItsBeno** [comment]
Cool!

**Comment by jspenguin2017** [comment]
I need to make a framework that would scrape this... This can take some time since I need to request each of these pages and extract the `iframe`. 
![image](https://cloud.githubusercontent.com/assets/7283682/26280300/5ce9fdb0-3d8a-11e7-8872-da05a5c52df8.png)

**Comment by ItsBeno** [comment]
Thank you @jspenguin2017  this might be useful 👍

**Comment by jspenguin2017** [comment]
They actually don't have that many domains 😕 
![image](https://cloud.githubusercontent.com/assets/7283682/26287363/34bd3b24-3e37-11e7-8bbd-a913e9dece4b.png)
```
albtvhd.com,tvshqiphd.com,www.balkanweb.com,cdn.livestream.com,livestream.artmotion.al:8080,shqip-tv.info
```

The complete log is here: 
https://github.com/jspenguin2017/AdBlockRadar/blob/master/!Tasks/shqiptvlive_net/Final%20Result%202017-4-21%2014-46-28.txt

**Comment by jspenguin2017** [comment]
OK, the automated process didn't fully work... We need to read over the log... 
https://github.com/jspenguin2017/AdBlockRadar/blob/master/!Tasks/shqiptvlive_net/Final%20Result%202017-4-21%2014-46-28.txt

**Comment by jspenguin2017** [comment]
NVM Those pages actually don't have `iframe`, they use Flash...
